canonical/vanilla-framework
# Sliding animation wip

## Done

Updated @pedoch 's branch to try and match the animation @juanruitina requested

Don't merge, this is just for the demo

Fixes [list issues/bugs if needed]

## QA

- Open [demo](insert-demo-url)
- reduce the viewport size
- click on nesting element to see the animation
